### DISCUSSION TITLE
Need to override twig file

### DISCUSSION THREAD

**Discussion by baladpi** [pimcore/pimcore#17516] [Q&A / Support] [answered]
I would like to override /var/www/pimcorefolder/vendor/pimcore/customer-management-framework-bundle/src/Resources/views/admin/duplicates/list.html.twig need to be override in custom bundle e.g : /var/www/pimcorefolder/bundles/CustomBundle/Resources/views/CustomerManagementFrameworkBundle/admin/duplicates/list.html.twig  

But I create the file, but override is not working, please help me

**Comment by jdreesen** [comment]
https://symfony.com/doc/6.4/bundles/override.html#templates

**Reply by baladpi** [reply]
Thank You but not working

Original Path: /vendor/pimcore/customer-management-framework-bundle/src/Resources/views/admin/duplicates/list.html.twig
Custom Override Path: /bundles/MyBundle/Resources/views/CustomerManagementFrameworkBundle/admin/duplicates/list.html.twig

**Reply by baladpi** [reply]
twig:
    paths:
        '%kernel.project_dir%/bundles/MyBundle/Resources/views': CustomerManagementFrameworkBundle

add this in twig.yaml also

**Comment by baladpi** [comment, marked as answer]
twig:
    paths:
        '%kernel.project_dir%/bundles/MyBundle/Resources/views/CustomerManagementFrameworkBundle': PimcoreCustomerManagementFramework

Add in twig.yaml


PimcoreCustomerManagementFramework  - namespace of the bundle  

php bin/console debug:twig  - get the namespace of the bundle

If not working : 
Try : https://stackoverflow.com/questions/50605939/override-bundle-template-from-another-bundle-in-symfony-4-5
